deploy/install_taxbrain_server.sh

```diff
@@ -67,7 +67,7 @@ if prompt_user "Install conda requirements?"; then
     fi
   done
   # Places these packages last, to install taxcalc, etc. at the end.
-  PACKAGES+=('btax' 'ogusa')
+  PACKAGES+=('btax' 'ogusa' 'taxcalc')
   echo "conda install $CHANNEL ${PACKAGES[@]}"
```

Comment:
Can you still specify which version of these packages that we want to use?

Reply:
You could do that after the script runs, or by changing the script. I guess right now it just gets the most recent one available.

Reply:
We need to be able to specify the version in the script. It's pretty common to have to pin to an older version. Each parameter change and API change requires a change on the PolicyBrain side. These tend to happen a lot. And on occasion they break something on the PolicyBrain side that we don't have time to solve at the moment.